src/components/Card/Card.js

```diff
@@ -86,9 +86,9 @@ const channelsList = channels => {
   });
 };
 
-const Avatar = ({ mentor, id }) => {
+const Avatar = ({ mentor, id, handleAvatarClick }) => {
   return (
-    <div className="avatar">
+    <div className="avatar" onClick={handleAvatarClick}>
       <i className="fa fa-user-circle" />
```

Comment:
Consider using `a` tag for accessibility.

Reply:
changes implemented in the latest commit.